In POSIX sh, [[ ]] is undefined.

[out script](https://github.com/homedepot/newrelic-deploy-resource/blob/master/out)
    - Lines 23, 26

[SC2039](https://github.com/koalaman/shellcheck/wiki/SC2039): In POSIX sh, [[ ]] is undefined.

Note: [Bash supports [[ ]]](http://mywiki.wooledge.org/BashFAQ/031)
This was [originally written in Bash](https://github.com/homedepot/newrelic-deploy-resource/commit/5e2a1bc27f83963d6cc6d7385e179ff5771d0d66#diff-c68271a63ddbc431c307beb7d2918275) and switched to shell (i.e. files renamed without changes)

Source: [ShellCheck](https://www.shellcheck.net/)

--- a/out
+++ b/out
@@ -20,11 +20,8 @@ json_file=$(jq -r '.params.json_file // ""' < $inputs)
 user=$(jq -r '(.params.user // empty)' < $inputs)
 
 
-if [[ -n "${json_file}" && -f "${json_file}" ]]
-then
-  check=$(jq '.revision?' $json_file)
-  if [[ $check == null ]]
-  then
+if [ -n "${json_file}" ] && [ -f "${json_file}" ]; then
+  if [ "$(jq 'has("revision")' $json_file)" = "false" ]; then
     echo "revision is missing from the json_file. cannot continue"
     exit 1
   fi

